src/app/components/Search/index.tsx

```diff
@@ -220,67 +225,66 @@ const SearchCmp: FC<SearchProps> = ({ scope, variant, disabled, onFocusChange: o
         placeholder={searchPlaceholderTranslated}
         fullWidth
         FormHelperTextProps={{
+          // turn off TS temporary for next line
+          // issue https://github.com/mui/material-ui/issues/33339
+          // fix https://github.com/mui/material-ui/pull/35924
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          // @ts-ignore
```

Comment:
ts-expect-error does not need lint override

and from docs
```
Pick ts-expect-error if:
...
- you expect a fix to be coming in fairly quickly and you just need a quick workaround
```

Reply:
Indeed. Changed accordingly.